scripts/retrieve_credits.rb

```diff
@@ -0,0 +1,219 @@
+# frozen_string_literal: true
+
+require 'json'
+require 'nokogiri'
+require 'open-uri'
+require 'date'
+
+require_relative '../RubySetupSystem/RubyCommon'
+
+require_relative 'po_helpers'
+require_relative 'json_helpers'
+require_relative 'folder_detector'
+
+DEVELOPERS_PAGE = 'https://wiki.revolutionarygamesstudio.com/wiki/Team_Members'
+DONATIONS_PAGE = 'https://wiki.revolutionarygamesstudio.com/wiki/Donations'
+
+# +31 is here to guarantee donations made in the last day of a month will be included properly
+DONATION_DISPLAY_CUTOFF = (365 + 31) * 60 * 60 * 24
+
+FILE_DOWNLOAD_OLDNESS = 60 * 60 * 24
```

Comment:
age?

Reply:
Replacing "oldness" with "age" doesn't actually make the name descriptive. This is the threshold when to consider a file old. Now that I'm looking at this I'm wondering why I didn't add the word "threshold" in there.

Reply:
Made a commit where I renamed it, and it now includes the word "age" in it.